tests/login.go

```diff
@@ -19,7 +19,11 @@ import (
 // LoginToCirros call LoggedInFedoraExpecter but does not return the expecter
 func LoginToCirros(vmi *v1.VirtualMachineInstance) error {
 	expecter, err := LoggedInCirrosExpecter(vmi)
-	defer expecter.Close()
+	defer func() {
```

Comment:
Great catch. I think that in this case we don't need a defer at all. Maybe something like this:

```golang
if err == nil {
   expecoter.Close()
}
return err
```

is sufficient in this case.

Reply:
Thanks a lot @rmohr! <3 

I agree that the defer is not needed, just pushed changes but instead of checking for the error, checking for the expecter not being nil, so that we don't assume that err == nil => expecter != nil, let me know wdyt